R/Backend-class.R

```diff
@@ -270,3 +274,27 @@ setGeneric(
         standardGeneric("backendWriteSpectra"),
     valueClass="Backend"
 )
+
+#' Subset the `Backend` by spectra (`i`) and/or file (`file`).
```

Comment:
This documentation is now outdated. Could you please adapt it to the new implementation.

Reply:
Will do.

Reply:
Done